client/my-sites/stats/activity-log/threat-alert.jsx

```diff
@@ -122,19 +124,50 @@ export class ThreatAlert extends Component {
 					className="activity-log__threat-alert"
 					highlight="error"
 					compact
+					clickableHeader={ true }
+					actionButton={ <span /> }
 					header={
 						<Fragment>
 							<ActivityIcon activityIcon="notice-outline" activityStatus="error" />
 							<div className="activity-log__threat-alert-header">
-								<div>
+								<div className="activity-log__threat-header-top">
 									<span className="activity-log__threat-alert-title">
 										{ headerTitle( translate, threat ) }
+										<TimeSince
+											className="activity-log__threat-alert-time-since"
+											date={ threat.firstDetected }
+											dateFormat="ll"
+										/>
 									</span>
-									<TimeSince
-										className="activity-log__threat-alert-time-since"
-										date={ threat.firstDetected }
-										dateFormat="ll"
-									/>
+									<SplitButton
+										compact
+										primary
+										label={ translate( 'Fix threat' ) }
+										onClick={ () => console.log( 'main button clicked' ) }
```

Comment:
Hm, this causes CI tests to fail, see https://circleci.com/gh/Automattic/wp-calypso/95822. Could you remove the console.log statements in a follow-up?